code/game/mecha/mecha_wreckage.dm

```diff
@@ -197,6 +197,10 @@
 			welder_salvage += part
 			parts -= part
 
+/obj/structure/mecha_wreckage/aquifer
+	name = "\improper Aquifer wreckage"
+	icon_state = "aquifer-broken"
+	desc = "Return to nothing... Just kidding, this shit is plastic."

```

Comment:
The aquifer is no longer made of plastic...

My immersion is RUINED, smh...

Reply:
Yknow don't, say swears

Reply:
Right. I'm sorry.